.vnurc

```diff
@@ -11,4 +11,3 @@ Attribute “aria-setsize” not allowed on element “tr” at this point.
 # Ignoring role meter
 Bad value “meter” for attribute “role” on element “div”.
 # Deleted Section Archive
-The “longdesc” attribute on the “img” element is obsolete. Use a regular “a” element to link to the description.
```

Comment:
I removed this one to show the build failures, but I'm wondering if the `aria-practices-DeletedSectionsArchive.html` shouldn't just be removed from the linting

Reply:
yes, no reason to lint the deleted sections file

Reply:
Sounds good. I can submit a separate PR for that, since it will just change the commandline in package.json rather than change anything in this PR